src/components/LegacyOrderableList/LegacyOrderableList.stories.tsx

```diff
@@ -0,0 +1,63 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Meta, StoryFn } from '@storybook/react';
+import { useState } from 'react';
+import { LegacyOrderableList as OrderableListComponent } from './LegacyOrderableList';
+import { LegacyOrderableListItem } from './types';
+import { LegacyOrderableListProps } from '.';
+import { chain } from '@react-aria/utils';
+import { renderContent, storyItems } from '@components/LegacyOrderableList/utils';
+
+export default {
+    title: 'Components/Legacy Orderable List',
```

Comment:
You could move this to `Deprecated` category in Storybook -> `Deprecated/Legacy OrderableList`